src/common/ceph_mutex.h

```diff
@@ -14,7 +14,7 @@
 // and make_recursive_mutex() factory methods, which take a string
 // naming the mutex for the purposes of the lockdep debug variant.
 
-#if defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
+#ifdef WITH_SEASTAR
```

Comment:
IIUC the alienized BlueStore would now use the `dummy_mutex` as e.g. `ceph::mutex`. I don't think it's ready for that.

Reply:
Alienized Bluestore would not change. Any component used by alien is no longer is defined `WITH_SEASTAR`, meaning in this case for example, Alienized Bluestore would still use real mutex.

Reply:
Sounds good!